src/pages/Dashboard/PortfolioBreakdown.tsx

```diff
@@ -56,11 +58,18 @@ export const PortfolioBreakdown = () => {
   //FOXY, OSMO, COSMO, Yarn Vaults
   const balances = useEarnBalances()
   //FOX/ETH LP Balance
-  const opportunity = useAppSelector(s => selectFoxEthLpAccountsOpportunitiesAggregated(s, null))
-  const lpBalance = opportunity?.underlyingFoxAmount ?? 0
+
+  const foxEthLpOpportunity = useAppSelector(state =>
+    selectAggregatedEarnUserLpOpportunity(state, {
```

Comment:
memo filter, no need for `?? ''` on `assetId`

Reply:
362a018729fcc44320cf1644b3fbe67596b9b4a5